flameshot.pro

```diff
@@ -198,24 +198,26 @@ RESOURCES += \
 
 # installs
 unix:!macx {
-    packaging {
-        USRPATH = /usr
-    } else {
-        USRPATH = /usr/local
+    isEmpty(PREFIX) {
+      packaging {
+        PREFIX = /usr
+      } else {
+        PREFIX = /usr/local
+      }
     }
 
     target.path = $${BASEDIR}$${USRPATH}/bin/
 
-    qmfile.path = $${BASEDIR}/usr/share/flameshot/translations/
+    qmfile.path = $${BASEDIR}$${USRPATH}/share/flameshot/translations/
     qmfile.files = $${TRANSLATIONS_FILES}
 
-    dbus.path = $${BASEDIR}/usr/share/dbus-1/interfaces/
+    dbus.path = $${BASEDIR}$${USRPATH}/share/dbus-1/interfaces/
     dbus.files = dbus/org.dharkael.Flameshot.xml
     
-    icon.path = $${BASEDIR}$${USRPATH}/share/icons/
+    icon.path = $${BASEDIR}$${USRPATH}/share/icons/hicolor/scalable/apps
```

Comment:
Not sure if scalable is the correct directory. But when I put the icon in .../hicolor/100x100/apps/, it didn't show up under KDE.